src/Task/Psalm.php

```diff
@@ -26,6 +26,7 @@ public function getConfigurableOptions(): OptionsResolver
             'ignore_patterns' => [],
             'no_cache' => false,
             'report' => null,
+            'output_format' => 'compact',
```

Comment:
Can you set the default value to null? That way it uses the default 'console' output from psalm, but makes it possible to overwrite in your local grumphp.yaml config.

https://github.com/vimeo/psalm/blob/master/src/psalm.php#L211-L213